pom.xml

```diff
@@ -45,7 +45,7 @@
 		<uilibraryVersion>2.0.5-SNAPSHOT</uilibraryVersion>
 		<calculationVersion>1.3-SNAPSHOT</calculationVersion>
 		<serialization.xstreamVersion>0.2.12-SNAPSHOT</serialization.xstreamVersion>
-		<htmlwidgetsVersion>1.7.3-SNAPSHOT</htmlwidgetsVersion>
+		<htmlwidgetsVersion>1.7.2-SNAPSHOT</htmlwidgetsVersion>
```

Comment:
@taremwatadeo snapshot is only for the latest development version. Once released, snapshot is dropped. This is why travis reports a failure for your pull request.